### PR TITLE
Brightcove player id analytics

### PR DESCRIPTION
**Pull request by xahgmah**
Add player id to studio form.
 Consider player id in student view.
 Consider player id in brightcove.css
https://trello.com/c/GFHQUnnq/58-0-5-improvement-brightcove-player-id-analytics

### CODE REVIEW

File: video_xblock/static/css/brightcove.css [hunk 1]
```diff
@@ -327,3 +327,52 @@ body {
     background: rgba(115,133,159,.5);
     border-radius: 0;
 }
+
+
+/* This styles are depended to player id */
```
Comment (polesye) [inline video_xblock/static/css/brightcove.css]: How do they depends on player id?

Comment (polesye) [inline video_xblock/static/css/brightcove.css]: `The following styles depends on player id.`

File: video_xblock/static/css/brightcove.css [hunk 2]
```diff
@@ -327,3 +327,52 @@ body {
...
+.bc-player-{{ player_id }}_default .vjs-menu-button-popup .vjs-menu .vjs-menu-item {
+    font-weight: bold;
+}
+
+.bc-player-{{ player_id }}_default .vjs-menu-button-popup .vjs-menu .vjs-menu-item:hover {
+    text-shadow: none;
+    color: inherit;
+    background: rgba(115, 133, 159, .5);
+    border-radius: 0;
+}
```
Comment (polesye) [inline video_xblock/static/css/brightcove.css]: add empty line

File: video_xblock/static/css/brightcove.css (hunk 1 above)
Comment (xahgmah) [inline video_xblock/static/css/brightcove.css]: Player id hardcoded to css classes of player

**Comment by polesye** [comment]
@madeira please review

**Comment by polesye** [comment]
@xahgmah please fix conflicts

**Comment by polesye** [comment]
@xahgmah don't forget to cleanup commit history before merge

**Pull request merged**